.travis.yml

```diff
@@ -6,13 +6,28 @@ php:
   - 5.6
   - 7
 
+env:
+   - SYMFONY_VERSION=2.3.*
+   - SYMFONY_VERSION=2.7.*
+   - SYMFONY_VERSION=2.8.*
+   - SYMFONY_VERSION=3.0.*
+
+matrix:
+  exclude:
+    - php: 5.4
+      env: SYMFONY_VERSION=3.0.*
```

Comment:
+1
Symfony3 is not compatible with PHP 5.4, that's why I exclude this test.
Does Symfony 2.3 supports still interesting ? It's end of maintenance raises in 2 months...